### PR TITLE
ThresholdAlerts feature

### PR DESCRIPTION
**Pull request by emarsonayap12**

### CODE REVIEW

File: ThresholdAlerts.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+fun CheckThresholds(data: pollutant_data): List<String> {
```
Comment (josechancelonaabaya) [inline ThresholdAlerts.kt:2]: // Non-compliant: Poor function name, missing comments, logic bug introduced

Comment (Chancelon) [inline ThresholdAlerts.kt]: Non-compliant: Poor function name, missing comments, logic bug introduced

File: ThresholdAlerts.kt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    val alerts = mutableListOf<String>()
+    if (data.pm25 > 35) alerts.add("High PM2.5!")
```
Comment (josechancelonaabaya) [inline ThresholdAlerts.kt:4]: No spaces

File: ThresholdAlerts.kt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+fun CheckThresholds(data: pollutant_data): List<String> {
+    val alerts = mutableListOf<String>()
+    if (data.pm25 > 35) alerts.add("High PM2.5!") 
+    if (data.No2 > 75) { alerts.add("High NO2!") }
```
Comment (josechancelonaabaya) [inline ThresholdAlerts.kt:5]: Missing edge case check

File: ThresholdAlerts.kt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+
+fun CheckThresholds(data: pollutant_data): List<String> {
+    val alerts = mutableListOf<String>()
+    if (data.pm25 > 35) alerts.add("High PM2.5!") 
+    if (data.No2 > 75) { alerts.add("High NO2!") } 
+    if (data.CO > 20) { alerts.add("High CO!") }
+    if (data.CO < 0 || data.No2 < 0 || data.pm25 < 0) alerts.add("Negative values are not allowed") // Unhandled scenario
```
Comment (Chancelon) [inline ThresholdAlerts.kt]: // Unhandled scenario

File: ThresholdAlerts.kt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+
+fun CheckThresholds(data: pollutant_data): List<String> {
+    val alerts = mutableListOf<String>()
+    if (data.pm25 > 35) alerts.add("High PM2.5!") 
+    if (data.No2 > 75) { alerts.add("High NO2!") } 
+    if (data.CO > 20) { alerts.add("High CO!") }
+    if (data.CO < 0 || data.No2 < 0 || data.pm25 < 0) alerts.add("Negative values are not allowed")
```
Comment (Chancelon) [inline ThresholdAlerts.kt]: Unhandled scenario

**Review by Chancelon — COMMENTED** [review]
Add proposed changes

**Pull request merged**